### DISCUSSION TITLE
Multicalss Focal Loss

### DISCUSSION THREAD

**Discussion by RightL** [Project-MONAI/MONAI#6508] [Ideas]
**Is your feature request related to a problem? Please describe.**
I notice that the focal loss implementation is currently using the BCE loss and sigmoid activation function. For the multiclass situation, would it be more appropriate to use softmax instead?

**Comment by wyli** [comment]
thanks, I create a ticket to track the feature request details https://github.com/Project-MONAI/MONAI/issues/6510

please feel free to add more info
